java-security/src/main/java/com/sap/cloud/security/token/validation/validators/JwtAudienceValidator.java

```diff
@@ -94,6 +93,15 @@ private ValidationResult validateAudienceOfXsuaaBrokerClone(Set<String> allowedA
 				audiences.add(audience);
 			}
 		}
+		// extract audience (app-id) from scopes
+		if (audiences.size() == 0) {
```

Comment:
It would be more idiomatic to use `isEmpty` here

Reply:
Done